python/raydp/services.py

```diff
@@ -1,109 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-from abc import ABC, abstractmethod
-from typing import Any, Dict, NoReturn
-
-
-class Cluster(ABC):
```

Comment:
I believe this is for supporting multiple big data frameworks in RayDP, not just for Spark. There are still possibility for that. So should we keep this? cc @ConeyLiu

Reply:
That's right. I think we should keep it left if we want to support more distributed frameworks. Some other frameworks (such as flink, presto) are all popular and widely used in many companies.